src/Divider/Divider.js

```diff
@@ -22,7 +23,12 @@ export const styleSheet = createStyleSheet('Divider', (theme) => {
   };
 });
 
-export default function Divider(props, context) {
+type Props = {
+  absolute?: boolean,
+  className?: string,
+};
+
+export default function Divider(props:Props, context:{styleManager: Object}):Element {
```

Comment:
I'm just following the linter, it may be silent on this but if this is a desire, we should try to enforce it through that.